src/coreclr/vm/method.hpp

```diff
@@ -723,6 +723,7 @@ class MethodDesc
     inline PTR_DynamicMethodDesc AsDynamicMethodDesc();
     inline bool IsDynamicMethod();
     inline bool IsILStub();
+    inline bool IsDiagnosticsHidden();
     inline bool IsLCGMethod();
 
```

Comment:
```suggestion
    inline bool IsLCGMethod();

    inline bool IsDiagnosticsHidden();
    
```
Nit: Keep the dynamic methoddescs related methods together